MAINTAINERS_RULES.md

```diff
@@ -0,0 +1,39 @@
+# Maintainers Rules
+
+This document lays out some basic rules and guidelines all maintainers are expected to follow.
+Changes to the [Acceptance Criteria](#hard-acceptance-criteria-for-merging-a-pr) for merging PRs require a ceiling(two-thirds) supermajority from the maintainers.
+Changes to the [Repo Guidelines](#repo-guidelines) require a simple majority.
+
+## Hard Acceptance Criteria for merging a PR:
+
+- 2 LGTMs are required when merging a PR
+- If there is obviously still discussion going on in the PR, even with 2 LGTMs, let the discussion resolve before merging. If you’re not sure, reach out to the maintainers involved in the discussion.
+- All checks must be green 
+    - There are limited mitigating circumstances for this, like if the docs builds are just broken and that’s the only test failing.
+    - Adding or removing a check requires simple majority approval from the maintainers.
+
+## Repo Guidelines:
```

Comment:
should this section go in the contributing.md? This seems more like a statement for general developers rather than specific to maintainers

Reply:
yes probably.